AsyncDisplayKit/ASMultiplexImageNode.h

```diff
@@ -69,6 +69,11 @@ typedef NS_ENUM(NSUInteger, ASMultiplexImageNodeErrorCode) {
 @property (nonatomic, readwrite, assign) BOOL downloadsIntermediateImages;
 
 /**
+ * @abstract Indicates that the receiver should stop loading if it encounters an error while loading an intermediate image identifier. Defaults to NO.
+ */
+@property (nonatomic, readwrite, assign) BOOL haltLoadingOnError;
```

Comment:
nit:  let's name this `haltsLoadingOnError`, to match `downloadsIntermediateImages`